src/connector/src/parser/unified/mod.rs

```diff
@@ -0,0 +1,43 @@
+//! Unified parsers for both normal events or CDC events of multiple message formats
+
+use risingwave_common::types::{DataType, Datum};
+use thiserror::Error;
+
+pub mod avro;
+pub mod debezium;
+pub mod json;
+pub mod upsert;
+pub mod util;
+
+pub type AccessResult = std::result::Result<Datum, AccessError>;
+
+/// Access a certain field in an object according to the path
+pub trait Access {
```

Comment:
The name `Access` is too general. maybe narrow to something related to the parser.

Reply:
This trait is under the namespace `parser::unified`.